src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -909,18 +903,24 @@ bool CommandRequestImpl::IsMobileResultSuccess(
       mobile_apis::Result::SAVED);
 }
 
+bool CommandRequestImpl::IsHMIResultSuccess(
+    hmi_apis::Common_Result::eType result_code) {
```

Comment:
@mkorniichuk const to parameter?

Reply:
@AByzhynar fixed in 5db179eb26efd1a99c0178db3afb9d1e077cb383